stat/distuv/poisson_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv
+
+import (
+	"testing"
+
+	"gonum.org/v1/gonum/floats"
+)
+
+func TestPoisson(t *testing.T) {
```

Comment:
With random numbers generated, we also can use the rest of the testing capabilities here.

Reply:
Ping on the extra tests to match the other variables, i.e. `testChiSquared`